### ISSUE TITLE
V10.12 reword

### ISSUE THREAD

**Issue by kravietz** [OWASP/ASVS#59]
V10.12 is worded in a quite confusing way and it's not clear what should be submitted and where to. I would reword it into:

*Verify that production website URL has been submitted to [preloaded list of Strict Transport Security domains](https://www.chromium.org/hsts) maintained by web browser vendors.*

Link https://www.chromium.org/hsts as it's maintained by Chromium now, but it's an obviously ad-hoc solution so I suggest the wording is generic enough to keep up with future organisational changes.

**Comment by vanderaj** [comment]
Resolved
